src/field_theory/chevalley_warning.lean

```diff
@@ -147,7 +147,17 @@ begin
     ... ≤ (q - 1) * (f i).total_degree : total_degree_pow _ _
 end
 
-/-- The Chevalley–Warning theorem.
+/-- The **Chevalley–Warning theorem**, fintype version.
+Let `(f i)` be a finite family of multivariate polynomials
+in finitely many variables (`X s`, `s : σ`) over a finite field of characteristic `p`.
+Assume that the sum of the total degrees of the `f i` is less than the cardinality of `σ`.
+Then the number of common solutions of the `f i` is divisible by `p`. -/
+theorem char_dvd_card_solutions_family' (p : ℕ) [char_p K p] [fintype ι] {f : ι → mv_polynomial σ K}
```

Comment:
I wonder if `char_dvd_card_solutions_of_sum_lt`,  `char_dvd_card_solutions_of_fintype_sum_lt`, and `char_dvd_card_solutions_of_add_lt` would be better names?

Reply:
I like them